panacea/src/pml_parser.yrl

```diff
@@ -169,6 +191,45 @@ operator -> '>=' : ">=".
 
 Erlang code.
 
+validate_time({TimeType, [{line, Line}], Val}) ->
```

Comment:
Anyone know of a nicer way to write this function? My Erlang is in the very early stages of learning so I'm not exactly sure what the options are..